Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift

```diff
@@ -462,13 +462,17 @@ struct CommunityView: View
             }
             .buttonStyle(EmptyButtonStyle()) // Make it so that the link doesn't mess with the styling
             .task {
-                if post == postTracker.posts.last {
-                    await loadFeed()
+                if !postTracker.isLoading {
+                    if let position = postTracker.posts.lastIndex(of: post) {
+                        if  position >= (postTracker.posts.count - 40) {
```

Comment:
the threshold here could likely be a bit closer to the bottom of the feed, but again I don't think it needs to change for this PR.

The API supports the ability to [mark posts as read](https://join-lemmy.org/api/interfaces/MarkPostAsRead.html), so we may want to look at having the post tracker be informed when posts are viewed to support that feature, at which point it could have the responsibility for deciding when the load the next page to remove this logic from the view all together.

Again nothing to change here, I'm just thinking out loud 🙃